devstack/Makefile

```diff
@@ -0,0 +1,34 @@
+DEVSTACK_COMPOSE_FILES=-f devstack/devstack-compose-ocim.yml.tmp
+
+ifeq ($(firstword $(MAKECMDGOALS)),$(filter $(firstword $(MAKECMDGOALS)),devstack.shell))
```

Comment:
These lines are a little hard to read and know what they are doing.

Reply:
I could not agree more :-) I'll remove them.